<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputDirectoryLineEdit">
        <property name="toolTip">
         <string>Directory that contains raw images</string>
        </property>
        <property name="placeholderText">
         <string>Input directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="inputDirectoryButton">
        <property name="text">
         <string>Open..</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputDirectoryLineEdit">
        <property name="toolTip">
         <string>Directory where to output converted images</string>
        </property>
        <property name="placeholderText">
         <string>Output directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outputDirectoryButton">
        <property name="text">
         <string>Open..</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startConversionButton">
      <property name="text">
       <string>Start Conversion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
